models/classes/LtiProvider/Form/LtiProviderForm.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019-2020 (original work) Open Assessment Technologies SA
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLti\models\classes\LtiProvider\Form;
+
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\service\ServiceManager;
+use oat\tao\model\controller\SignedFormInstance;
+use oat\taoLti\models\classes\LtiProvider\LtiProviderValidationService;
+use oat\taoLti\models\classes\LtiProvider\RdfLtiProviderRepository;
+use tao_helpers_Uri;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class LtiProviderForm extends SignedFormInstance
```

Comment:
IMO We do not need this class https://github.com/oat-sa/extension-tao-lti/pull/274/files#r490749187